test/e2e/data/cluster-templates/docker-rke2.yaml

```diff
@@ -17,10 +17,10 @@ spec:
   clusterNetwork:
     pods:
       cidrBlocks:
-      - 10.45.0.0/16
+      - 10.1.0.0/16
```

Comment:
Thanks for updating this, I see the change follows in CAAPF steps, which is great. This range also does not overlap with anything for my machine.